### DISCUSSION TITLE
"@" Functionality

### DISCUSSION THREAD

**Discussion by crendrik** [docmost/docmost#288] [Ideas]
Similar to Notion, it would be interesting to have certain functions when pressing "@":
- mention a user
- date integration 
- link to another page

![image](https://github.com/user-attachments/assets/6034b241-f64d-487f-a2d8-854a232eba51)

**Comment by jaygz316** [comment]
This would be amazing. Having switched from Trillium, this is the only feature I really miss.

**Reply by Philipinho** [reply]
It’s under development.
![image](https://github.com/user-attachments/assets/34bd17fd-dfef-413d-81cd-6851b9f958fc)

**Reply by jaygz316** [reply]
Dude no way, you just made my day. Looks like it's going to be a good Christmas. Happy holidays!

**Reply by challapradyumna** [reply]
@Philipinho Which branch is this in ?

**Comment by mustafa-yilmaz** [comment]
This feature should also be available (usable) in table cells, as also expressed at #655 .

**Comment by challapradyumna** [comment]
Leaving this here for others to track. https://github.com/docmost/docmost/pull/604

**Comment by mustafa-yilmaz** [comment]
It is nice to see this feature also works in table cells (with the "v0.8.0" release [here](https://github.com/docmost/docmost/releases/tag/v0.8.0) ).

**Comment by jonock** [comment]
Could we close this as this is now implemented?
